pyk/src/pyk/kast/outer_parser.py

```diff
@@ -120,8 +125,17 @@ def module(self) -> Module:
         while self._la.type is not TokenType.KW_ENDMODULE:
             sentences.append(self.sentence())
 
+        end_loc = self._la.loc + self._la.text
         self._consume()
 
+        location = (
+            (
+                Atts.LOCATION.name,
+                f'{begin_loc.line},{begin_loc.col},{end_loc.line},{end_loc.col}',
+            ),
+        )
+        source = ((Atts.SOURCE.name, self._source),) if self._source else ()
+        att = Att(att.items + location + source)
```

Comment:
My problem with this is now the text and the data structure are no longer one-to-one. This can be an issue for example if we want to use the parser for reading in K text as test data for kompiler pass unit tests.

Reply:
These attributes wont be added at parse time now, but are still going to be added to the `KAst` objects when converting from `AST`, would adding a flag to disable that be helpful?

Reply:
I can't think of a use case yet where not adding them would be necessary, let's just add them unconditionally for now.